### DISCUSSION TITLE
The large model is not fully displayed

### DISCUSSION THREAD

**Discussion by yygbruce** [google/filament#6849] [Q&A]
Hello, everyone! I load the model with the MeshAssimp class (filamentapp module's MeshAssimp.h and MeshAssimp.cpp),when the model position is large, cannot display the model fully.The screenshot:
![Screenshot from 2023-05-27 16-19-04](https://github.com/google/filament/assets/18020338/28331540-30bd-4241-ad43-a738e9177c51)
After i scale the position when load the model data to filament::VertexBuffer, it can display fully.The screenshot:
![Screenshot from 2023-05-27 17-05-16](https://github.com/google/filament/assets/18020338/12723f74-c7b9-4268-9c2b-5ca66600ef05)
And when i use gltfio library to load the model, it also work well.I try to scale the model with the class TransformManager, but it don't work. I set the camera's projection(30, double(w) / (h <= 0 ? 1 : h), 0.0001, 3000.0)).I want to know that how to scale the model like the gltfio library.I hope someone can give me some advice！Thanks!
[dragon_-_mystic_the_lost_ark.zip](https://github.com/google/filament/files/11581512/dragon_-_mystic_the_lost_ark.zip)

**Comment by romainguy** [comment]
You shouldn't set such a small near, use 0.01 for instance. What did you try with TransformManager that didn't work?

**Reply by yygbruce** [reply]
> You shouldn't set such a small near, use 0.01 for instance. What did you try with TransformManager that didn't work?

Thanks! Romain.I have soloved this problem.In the MeshAssimp.h, the half's range is too small, and when I change it with float, it work well.
![Screenshot_2023-05-28-11-29-52-984_org qtproject filament](https://github.com/google/filament/assets/18020338/53cc50b0-4e95-4a37-92cb-5f60e2004301)
